repobee_sanitizer/_sanitize_repo.py

```diff
@@ -63,21 +60,29 @@ def create_extension_command(self) -> plug.ExtensionCommand:
         """
         parser = plug.ExtensionParser()
         parser.add_argument(
+            "-r",
+            "--repo-root",
+            help="Path to the worktree root of the repository to sanitize.",
+            type=pathlib.Path,
+            metavar="path",
+            default=pathlib.Path("."),
+        )
+
+        files_mutex_grp = parser.add_mutually_exclusive_group(required=True)
+        files_mutex_grp.add_argument(
             "-f",
             "--file-list",
             help="Path to a list of files to sanitize. The paths should be "
             "relative to the root of the repository.",
             type=pathlib.Path,
             metavar="path",
-            required=True,
         )
-        parser.add_argument(
-            "-r",
-            "--repo-root",
-            help="Path to the worktree root of the repository to sanitize.",
-            type=pathlib.Path,
-            metavar="path",
-            default=pathlib.Path("."),
+        files_mutex_grp.add_argument(
+            "-d",
+            "--discover-files",
+            help="Find and sanitize all files in the repository that contain "
+            "at least one sanitizer marker.",
```

Comment:
Just a note here: Do we call them "sanitizer markers" in general? I'm not sure we have consistent terminology. We should probably decide on something.

Reply:
In the beginning, I called them blocks but realized that matched better with everything in between a start and an end marker. Since then ive tired at least to be consistent with calling them markers